src/installer/pkg/sfx/Microsoft.NETCore.App/Directory.Build.props

```diff
@@ -279,6 +279,7 @@
     <PlatformManifestFileEntry Include="libicuuc.a" IsNative="true" />
     <!-- zlib-specific files -->
     <PlatformManifestFileEntry Include="libz.a" IsNative="true" />
+    <PlatformManifestFileEntry Include="zlibstatic.lib" IsNative="true" />
```

Comment:
is it intentional to *ship* zlibstatic.lib in our shared framework?  That seems unusual as it's a dev-time resource - however I see other dev-time resources here like `main.c`, `driver.h`, etc.  I take it you needed to add this to build - but I bet a better fix is to get all of this stuff out of the platform manifest since I don't think it needs to be fed to the SDK for conflict resolution @jkoritzinsky @dsplaisted.

I see we have a metadata flag `ExcludeFromDataFiles` for that: https://github.com/dotnet/arcade/blob/76f733ee57811c38bb5b8e1ac9c6c50e92bc5dc9/src/Microsoft.DotNet.SharedFramework.Sdk/targets/sharedfx.targets#L249

I think we should review all the things added here, remove files that don't need to ship - if any, and apply `ExcludeFromDataFiles` for the files that need to ship and don't actually need to go into the manifest.  That can be done separately.

Reply:
I'll keep this in mind as a follow up.

Reply:
> is it intentional to ship zlibstatic.lib in our shared framework?

zlibstatic.lib ships in the native aot runtime package only. It does not ship in the regular shared framework.

Reply:
When I implemented the shared framework SDK, no one really knew what needed to go into the manifest, so it requires/lists all files. If we know what types of files need to be in it, I can update the SDK to only validate (for the template case) or include (for the generative case) files that matter.